lib/capistrano-db-tasks/database.rb

```diff
@@ -106,8 +106,13 @@ def dump_cmd_ignore_data_tables_opts
   class Remote < Base
     def initialize(cap_instance)
       super(cap_instance)
-      @config = @cap.capture("cat #{@cap.current_path}/config/database.yml")
-      @config = YAML.load(ERB.new(@config).result)[@cap.fetch(:rails_env).to_s]
+      @cap.within @cap.current_path do
+        @cap.with rails_env: @cap.fetch(:rails_env) do
+          @config = @cap.capture(:rails, 'runner "puts Rails.application.config.database_configuration[Rails.env].to_yaml"', '2>/dev/null')
+          @config.gsub!(/.*\-\-\-/m, '---') # Remove all bundler and rails initialization errors
+        end
```

Comment:
Nah.. Actually it does pretty much the same thing https://github.com/donbobka/capistrano-db-tasks/blob/77cbfcc5f5ff3a2be6f552cb54f8ae37294b6b72/lib/tasks/capistrano-db-tasks.rake.. Ignore what I said.. Was just wondering if adding this extra layer somehow took care of the errors without you having to manually..